cmd/flipt/main.go

```diff
@@ -199,6 +238,8 @@ func buildConfig() (*zap.Logger, *config.Config) {
 		logger.Warn("configuration warning", zap.String("message", warning))
 	}
 
+	logger.Debug("configuration source", zap.String("path", path))
```

Comment:
Nice one for adding this debug line. Could we move it up under line 199 after we determine the path.

My thinking is there are a number of failure cases that can happen between picking the path and this line. It would be good to help the user know when those failures occur where Flipt was attempting to load configuration from.

Reply:
@GeorgeMac Yeah that was my initial thought, but it looks like the logger isn't really initialized until later in the `buildConfig` function

Reply:
Ahh fair point. The config drives the logger.
Maybe you could add just another field to the `fatal` call on line 187 then.
e.g. a `zap.String("config_path", path)` field.
Just so in that situation we have the context on the fatal call.

Reply:
@GeorgeMac Yes definitely!